src/js/components/DateInput/DateInput.js

```diff
@@ -92,6 +98,16 @@ const DateInput = forwardRef(
     // when format and not inline, whether to show the Calendar in a Drop
     const [open, setOpen] = useState();
 
+    const openCalendar = useCallback(() => {
+      setOpen(true);
+      announce(formatMessage({ id: 'dateInput.enterCalendar', messages }));
+    }, [announce, formatMessage, messages]);
+
+    const exitCalendar = useCallback(() => {
```

Comment:
naming suggestion: `closeCalendar` to be consistent with state variable of `setOpen` (open/close)